src/app/missions/detail/detail.component.html

```diff
@@ -155,17 +155,62 @@ <h3>Rewards</h3>
 </section>
 
 <h3>Texts</h3>
+<ng-container *ngIf="mission && textsLocale">
+  <dl id="chat-bubbles">
+    <div *ngIf="textsLocale.chat_state_1">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_1" [innerHTML]="textsLocale.chat_state_1"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached before the mission is offered.</dd>
+    </div>
+    <div *ngIf="textsLocale.accept_chat_bubble">
+      <dt><blockquote class="chat-bubble" luxTooltip="accept_chat_bubble" [innerHTML]="textsLocale.accept_chat_bubble"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when the mission is accepted.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_2">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_2" [innerHTML]="textsLocale.chat_state_2"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached while the mission is in progress.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_3">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_3" [innerHTML]="textsLocale.chat_state_3"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached when the mission is ready to be completed.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_3_turnin">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_3_turnin" [innerHTML]="textsLocale.chat_state_3_turnin"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.target_objectID"></lux-object> when approached when the mission is ready to be completed.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_4">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_4" [innerHTML]="textsLocale.chat_state_4"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.offer_objectID"></lux-object> when approached when the mission is complete.</dd>
+    </div>
+    <div *ngIf="textsLocale.chat_state_4_turnin">
+      <dt><blockquote class="chat-bubble" luxTooltip="chat_state_4_turnin" [innerHTML]="textsLocale.chat_state_4_turnin"></blockquote></dt>
+      <dd>Spoken by <lux-object [luxFetch]="mission.target_objectID"></lux-object> when approached when the mission is complete.</dd>
+    </div>
+  </dl>
 
-<ng-container *ngIf="text">
-  <app-icon [iconID]="text.turnInIconID"></app-icon>
-</ng-container>
-
-<div *ngIf="textsLocale">
-  <ng-container *ngFor="let text of textsLocale | keys">
-    <h4 [id]="'text_' + text.key">{{text.key}}</h4>
-    <blockquote [innerHTML]="text.value"></blockquote>
+  <ng-container *ngIf="text">
```

Comment:
Why should this only be shown if `mission && textLocale` is true?

Reply:
The icon that is

Reply:
Ah right, I moved it down so it wouldn't mess with the layout, but forgot that there are additional conditions in this block. I'll move it out of there.